src/helpers/handleError.ts

```diff
@@ -24,6 +24,13 @@ export function parseError(error: unknown, defaultMessage = ErrorList.unknown) {
     const revertMessage = parseRevertReason(message)
     displayedError = gSNMessage || revertMessage || message
   }
+  if (
+    displayedError?.includes(
+      'can\'t access property "error", res2 is undefined'
```

Comment:
Why don't you use double quotes here? Like `"can't access..."`

Reply:
Here it seems better to take `can't access...`